install/rctf.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+# a script to manage rCTF installations
+
+import sys, os, io
+import argparse
+import logging
+
+
+logging.basicConfig(format = '%(message)s', level = logging.DEBUG)
+
+
+# define useful functions
+
+
+def verify_privileges(euid = 0):
+    if os.geteuid() != euid:
+        logging.fatal('This command must be run as root')
+        return False
+
+    return True
+
+
+def check_file(fn):
+    # verifies a file exists
+    return os.path.isfile(fn)
+
+
+def execute(command):
+    logging.debug('Executing `%s`...' % command)
+    
+    status_code = os.system(command)
+
+    if status_code:
+        logging.warning('Command failed to execute; exited with status code %d.' % status_code)
+
+    return status_code == 0
+
+
+# create simple model of rCTF
+
+
+REPOSITORY_BRANCH = 'master' # XXX: migrate REPOSITORY_BRANCH to an envvar/argument
+
+
+class rCTF:
+    def __init__(self, install_path = '/opt/rctf/'):
+        if not install_path.endswith('/'):
+            install_path += '/'
+
+        self.install_path = install_path
+
+
+    def up(self):
+        os.chdir(self.install_path)
+        
+        if not verify_privileges():
+            return False
+
+        if not execute('docker-compose up -d'):
```

Comment:
We might want to use `docker-compose up -d --build` to rebuild any changes that have been made - similarly for install script

Reply:
Thanks, that's a good point.

Reply:
fixed this!